cmd/bpf2go/main.go

```diff
@@ -275,6 +266,17 @@ func getEnv(key, defaultVal string) string {
 	return defaultVal
 }
 
+func getBoolEnv(key string, defaultVal bool) bool {
+	if val, ok := os.LookupEnv(key); ok {
```

Comment:
Nit: we prefer early returns for this kind of logic, e.g.:

```go
if val, ok := os.LookupEnv(key); !ok {
  return defaultVal
}
```

This makes the effect immediately follow the cause, and allows the rest of the function to be unindented.

Reply:
got it, thanks!